app/transport/http/resp.go

```diff
@@ -11,6 +11,12 @@ import (
 // that will be sent in JSON format
 type Response struct {
 	Message string `json:"message"`
+	Details string `json:"details"`
+}
+
+type CreateUserResponse struct {
+	Message string `json:"message"`
+	UserID  string `json:"id"`
 }
```

Comment:
I prefer storing specific responses near the according specific handlers, usually, they are used only with specific handlers and only in 1 file of Go code.
If you store response structures all in a separate file - you'll have to do more jumps over the code while reading it.

Reply:
I got it, thanks. Moved structs to one file with handlers